.github/workflows/add-to-release.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Add to Release'
+
+on: [release]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest 

```

Comment:
Windows pls

Reply:
Windows won't work, as you can see here: https://github.com/gruppe-adler/grad_meh/pull/2/commits/6b6514dc265f941c29b2d42f7e59c1086e4c785e